server/services/message-service.js

```diff
@@ -1,38 +1,32 @@
-const amqp = require('amqplib/callback_api')
-const scheduleService = require('./schedule-service')
+const container = require('rhea')
 const config = require('../config')
+const scheduleService = require('./schedule-service')
 
 module.exports = {
   receiveSchedule: function () {
-    const messageQueue = config.messageQueue
-    amqp.connect(messageQueue, function (err, conn) {
-      if (err) {
-        console.log(err)
-      } else {
-        conn.createChannel(function (err, ch) {
-          if (err) {
-            console.log(err)
-          } else {
-            const scheduleQueue = 'schedule'
-            ch.assertQueue(scheduleQueue, { durable: false })
-            console.log('waiting for new claims')
-
-            ch.consume(scheduleQueue, function (msg) {
-              console.log(`claim received for scheduling - ${msg.content.toString()}`)
-              scheduleService.create(JSON.parse(msg.content))
-            }, { noAck: true })
-
-            const valueQueue = 'value'
-            ch.assertQueue(valueQueue, { durable: false })
-            console.log('waiting for new calculation')
-
-            ch.consume(valueQueue, function (msg) {
-              console.log(`calculation received for payment - ${msg.content.toString()}`)
-              scheduleService.updateValue(JSON.parse(msg.content))
-            }, { noAck: true })
-          }
-        })
-      }
+    const scheduleQueue = 'schedule'
+    const valueQueue = 'value'

```

Comment:
the constants `scheduleQueue` and `valueQueue` could be put at the top of the file along with the `require` constants.

Other Defra services have taken the convention of having constants defined at the top of the file in uppercase with underscores, i.e. `SCHEDULE_QUEUE`, I think we should take the same approach.